hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -632,6 +635,15 @@ public static boolean isGCSFileSystem(FileSystem fs) {
     return fs.getScheme().equals(StorageSchemes.GCS.getScheme());
   }
 
+  /**
+   * Some filesystem(such as chdfs) will throw {@code IOException} instead of {@code EOFException}. It will cause error in isBlockCorrupted().
+   * Wrapped by {@code BoundedFsDataInputStream}, to check whether the desired offset is out of the file size in advance.
+   */
+  public static boolean shouldWrappedByBoundedDataStream(FileSystem fs) {
```

Comment:
can we keep it simple for now. 
```
  public static boolean isCHDSFileSystem(FileSystem fs) {
    return fs.getScheme().equals(StorageSchemes.CHDS.getScheme());
  }
```

if at all we come across other storage schemes which might need this, we can make it a map.